Makefile

```diff
@@ -52,7 +52,7 @@ exec: ## Run a command on the app container
 		echo "No command provided"; \
 		exit 1; \
 	fi; \
-	${DOCKER_COMPOSE} exec app $(command)
+	${DOCKER_COMPOSE} exec -T app $(command)
```

Comment:
What happens when there is a TTY? Should be extracted to a separate PR too.

Reply:
Strange that removing didn't affect the shell Make target. Will experiment.

Reply:
The prompt and niceties like completion disappear when running `sh` with `-T` (which means remove the TTY), so will want to keep the command normal for interactive use outside of CI.

Reply:
In https://github.com/libero/article-store/pull/77